client/src/app/containers/Game/ScoreBoard.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import {
+    Container,
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+} from '@material-ui/core';
+import PersonPinCircleIcon from '@material-ui/icons/PersonPinCircle';
+
+import { colorToHue } from '../../../utils/colorToHue';
+import { Team } from './models';
+
+// TODO abstract network models from the models we use so we can fix naming
```

Comment:
:mag: now or later?

Reply:
I'd prefer to get this PR in so we have somewhere to start. Happy to immediately follow up with a cleanup PR; just don't want this PR to be any more massive than it needs to be 🤷

Reply:
Open to your thoughts though if you feel strongly about including it

Reply:
love that. let's push it to later. "we're not under a time crunch"